lib/web/mage/sticky.js

```diff
@@ -11,7 +11,26 @@ define([
 
     $.widget('mage.sticky', {
         options: {
-            container: ''
+            container: '',
+            spacingTop: 0,
+            offsetTop: 0,
+            stickyClass: '_sticky'
+        },
+
+        _optionToNumber(option) {
```

Comment:
This is not correct function declaration. Move it out of widget in module private scope or declare this like method of widget.